### PR TITLE
add tuning guidelines

### PR DESCRIPTION
**Pull request by djfritz**

### CODE REVIEW

File: tuning/tuning.md [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+
+##### Prebuff-Sort-On-Consume
+
+Example: `Prebuff-Sort-On-Consume=true`
+
+The Prebuff-Sort-On-Consume parameter tells the prebuffer to sort locks of data prior to pushing them to disk. The sorting process is only applied to the individual block, and does NOT guarantee that data is sorted when entering the pipeline. Sorting blocks prior to storage also incurs a significant performance penalty in ingestion. Almost all installations should leave this value as false.
```
Comment (traetox) [inline tuning/tuning.md:197]: `locks` -> `blocks`

File: tuning/tuning.md [hunk 2]
```diff
@@ -0,0 +1,315 @@
...
+
+Example: `Disable-Compression=true`
+
+These parameters control user-mode compression of data in the wells. By default, Gravwell will compress data in the well. Setting `Disable-Hot-Compression` or `Disable-Cold-Compression` will disable it for the hot or cold storage, respectively; setting `Disable-Compression` disables it for both.
+
+##### Enable-Transparent-Compression, Enable-Hot-Transparent-Compression, Enable-Cold-Transparent-Compression
+
+Example: `Enable-Transparent-Compression=true`
+
+These parameters control kernel-level, transparent compression of data in the wells. If enabled, Gravwell can instruct the `btrfs` filesystem to transparently compress data. This is more efficient than user-mode compression. Setting `Enable-Transparent-Compression` true automatically turns off user-mode compression. Note that setting `Disable-Compression=true` will disable transparent compression.
```
Comment (traetox) [inline tuning/tuning.md:217]: `Transparent compression also has some other performance benefits as we can take advantage of some memory de-duplication, if you need the best possible performance from Gravwell combining transparent compression with a well tuned BTFS file system is the best way to achieve it.`

File: tuning/tuning.md [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+
+##### Acceleration
+
+As mentioned above, utilizing well acceleration can dramatically improve performance. See the [Acceleration documentation](#!configuration/accelerators.md) for more information.
+
+### Linux Kernel and System Tuning
+
+#### Disable swap
+
+If possible, disable swap support on any Gravwell indexer node. Swap helps alleviate memory pressure at the expense of writing data to disk. A busy Gravwell indexer is likely to create _thrashing_ in the swap subsystem.
```
Comment (traetox) [inline tuning/tuning.md:227]: is this true?  Is an OOM better than a slowdown?

**Review by traetox — CHANGES_REQUESTED** [review]
few nitpicks

**Comment by traetox** [comment]
would you also throw in `Ingest_Throttle_Threshold` to the global config parameters

File: tuning/tuning.md (hunk 3 above)
Comment (djfritz) [reply tuning/tuning.md]: I just removed this recommendation. A real recommendation would be to adjust vm.swapiness, but that has so many caveats...

**Comment by djfritz** [comment]
@traetox PTAL

**Review by traetox — APPROVED** [review]
LGTM

**Pull request merged**